telos-frontend/src/components/text/JournalText.jsx

```diff
@@ -1,3 +1,11 @@
-const JournalText = () => <div />;
+import Paper from '@material-ui/core/Paper';
+import styles from './TextWidget.module.css';
+
+// This react component draws a Material-UI paper background and overlays a text area on it for the user to make notes
+const JournalText = () => (
+  <Paper className={styles.paper} variant="outlined">
+    <textarea className={styles.textInput} />
```

Comment:
What was your reasoning for using the basic textarea element instead of a corresponding Material UI element?

Reply:
The corresponding Material-UI element was InputBase. This had functionality to grow the height of the text box when a new line was wanted, but wouldnt fit. This caused it to extend out of its parent container. I could limit the number of rows of text, but i cannot determine how many rows to limit to.

After considerable time trying to prevent the behaviour, I thought it was not a good use of time to continue. I decided to use the textarea as I could prevent it extending itself.

Reply:
Fair enough, good to know that you tried to keep it consistent